### DISCUSSION TITLE
Merging feature branches

### DISCUSSION THREAD

**Discussion by DerpSnepbek** [DerpSnepbek/Tafe24S1T112_Ext#4] [General]
Alright, so feature branch is all done.
![image](https://github.com/DerpSnepbek/Tafe24S1T112_Ext/assets/161171900/62dcd86d-48d6-4b72-a283-cfeda25d7e18)

We can go through merging the branches together this evening, if you're still happy to do so?

**Comment by Sunnyyu33a** [comment]
I didn't see this message and just done the merge... and... on my local repo, 
![image](https://github.com/DerpSnepbek/Tafe24S1T112_Ext/assets/123451322/dab3ef58-1ec8-4b84-96ff-3a0cb58fc8b9)
but!!! I pushed up and cloned it and check.... 
![image](https://github.com/DerpSnepbek/Tafe24S1T112_Ext/assets/123451322/b9681c9c-8830-4052-b4a7-1b2950720ff9)
I really hope we don't have to start over again............
